### DISCUSSION TITLE
[FEATURE]: Run a sale

### DISCUSSION THREAD

**Discussion by tman16** [etsy/open-api#733] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Describe the solution you'd like

I wish to request that it should be possible to create product sales via the api including scheduling.

- Create a sale and assign certain products
- To select the country the sale applies to
- Schedule the sale for a date in the future

**Comment by etsyachristensen** [comment]
Thank you for the suggestion. I'll add it to our list of items to discuss. This likely won't happen soon though, just to set expectations.

**Comment by etsyachristensen** [comment]
We're moving this to an Idea Discussion because we are closing the Issues board soon.

**Comment by JustJerem** [comment]
Any update on that API ? Thanks :)

**Comment by mhurkman** [comment]
I would love to see this feature. The website functionality to add products to a sale is bare-bones and inadequate.

**Comment by simonj** [comment]
Any update on this
